lib/driver.js

```diff
@@ -393,8 +393,19 @@ class EspressoDriver extends BaseDriver {
     // start an avd, set the language/locale, pick an emulator, etc...
     // TODO with multiple devices we'll need to parameterize this
     await helpers.initDevice(this.adb, this.opts);
+    this.caps.setAnimation = this.caps.setAnimation || false;
     // https://github.com/appium/appium-espresso-driver/issues/72
-    if (await this.adb.isAnimationOn()) {
+    if (this.caps.setAnimation && await this.adb.isAnimationOn()) {
```

Comment:
consider only calling `await this.adb.isAnimationOn()`. Each extra adb call takes time.